temporalio/lib/temporalio/internal/worker/workflow_worker.rb

```diff
@@ -13,7 +13,7 @@ module Internal
     module Worker
       # Worker for handling workflow activations. Most activation work is delegated to the workflow executor.
       class WorkflowWorker
-        def self.workflow_definitions(workflows)
+        def self.workflow_definitions(workflows, should_enforce_versioning_behavior)
```

Comment:
```suggestion
        def self.workflow_definitions(workflows, should_enforce_versioning_behavior:)
```

Internal code so doesn't matter much, but I recommend kwargs instead of positional for these kinds of trailing booleans